src/richie/apps/courses/models/organization.py

```diff
@@ -98,9 +98,12 @@ class OrganizationPluginModel(CMSPlugin):
     Organization plugin model handles the relation from OrganizationPlugin
     to their Organization instance
     """
-
-    organization = models.ForeignKey(Organization)
-
+    organization = models.ForeignKey(
+            "Organization",
+            related_name="organization_plugin",
```

Comment:
Much better but you don't need to include "organization" in the name because this is a related name so it is going to be called from an organization object. You will have to write: "organization.organization_plugin" which is redundant.

You will agree that "organization.plugin_model" is much better and makes more sense to the reader.